File: C/3170.c
#include <stdio.h>

int main()
{
    int bPossui, galhos, qtd, faltam;
    scanf("%d", &bPossui);
    scanf("%d", &galhos);
    
    qtd = galhos/2;
    
    faltam = qtd-bPossui;
    
    if(faltam>0){
        printf("Faltam %d bolinha(s)\n", faltam);
    } else{
        printf("Amelia tem todas bolinhas!\n");
    }
    
    
    return 0;
}

File: Python/3170.py
bolinhas = int(input())
galhos = int(input())

if galhos%2==0:
    qtd_bolinhas_faltantes = (galhos/2) - bolinhas
else:
    qtd_bolinhas_faltantes = (galhos-1)/2 - bolinhas
    
if qtd_bolinhas_faltantes>0:
    print(f"Faltam {int(qtd_bolinhas_faltantes)} bolinha(s)")
else:
    print("Amelia tem todas bolinhas!")